source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsController.cs

```diff
@@ -57,11 +58,36 @@ public ActionResult PeriodEvaluation(string period)
             //2.a. If not, redirect to /Evaluations/{period}/{username}
             //2.b. If so, fetch the users to evaluate
 
-            /// GK TESTING, PLEASE DELETE
-            //var employees = ExecuteCommand(new GetEmployeesForEvaluationCommand("2015-06"));
-            //ExecuteCommand(new GenerateEvaluationsCommand(employees));
+            var loggedUser = DetectUser();
 
-            //ExecuteCommand(new AddEvaluatorsCommand(employees[0], new List<string>() { "Users/gkolocsar" }));
+            RavenQueryStatistics stats;
+            IQueryable<EmployeeToEvaluate_Search.Projection> query = RavenSession
```

Comment:
Oh, I was searching for `query`, but I see that you are using `projection` variable later. Maybe it is possible to add a filter to the first query in order to avoid to bring a lot of results (potentially) from database.